### DISCUSSION TITLE
Mounting manually created cephfs volumes in kubernetes >=1.28.0

### DISCUSSION THREAD

**Discussion by adippl** [ceph/ceph-csi#4283] [Q&A]
# Describe the feature you'd like to have #

I need a way to mount large, manually created cephfs volume in k8s pod. 

Kubernetes 1.28.0 depreciates in tree CephFS mounting option and suggests using Ceph CSI driver instead for that use case.
Original mounting mechanism had ability to mount pre-existing cephfs volume to a pod. https://github.com/kubernetes/examples/blob/master/volumes/cephfs/cephfs.yaml
Ceph CSI driver seems to work only on dynamically provisioned modules. There are no examples on how to create PV manually.
https://github.com/ceph/ceph-csi/tree/devel/examples/cephfs

# What is the value to the end user? (why is it a priority?) #

Ability to mount manually created CephFS subvolumes

# Additional context #

I'm not sure if this use case is supported or not. It would be nice to have confirmation and example of how to create such PV.

**Comment by nixpanic** [comment]
Have a look at [the documentation about static provisioning](https://github.com/ceph/ceph-csi/blob/devel/docs/static-pvc.md#cephfs-static-pvc), I suspect that answers your question.

**Comment by adippl** [comment]
Yes, it helped. I missed this solution.
